UI RepositoryTestCase on RHEL6 should omit 'ostree'

Connected to #4040

Test to fix:
- [ ] [UI] RepositoryTestCase.test_negative_download_policy_displayed_for_non_yum_repo

Automation 6.2 results:
```

        if not matched:
>           raise NoSuchElementException("Could not locate element with visible text: %s" % text)
E           NoSuchElementException: Message: Could not locate element with visible text: ostree
```

--- a/tests/foreman/cli/test_repository.py
+++ b/tests/foreman/cli/test_repository.py
@@ -67,6 +67,7 @@ from robottelo.datafactory import (
     valid_http_credentials,
 )
 from robottelo.helpers import get_data_file
+from robottelo.host_info import get_host_os_version
 from robottelo.test import CLITestCase
 
 
@@ -597,9 +598,17 @@ class RepositoryTestCase(CLITestCase):
 
         @Assert: Non-YUM repository is not created with a download policy
         """
-        non_yum_repo_types = [
-            item for item in REPO_TYPE.keys() if item != 'yum'
-        ]
+        os_version = get_host_os_version()
+        # ostree is not supported for rhel6 so the following check
+        if os_version.startswith('RHEL6'):
+            non_yum_repo_types = [
+                item for item in REPO_TYPE.keys()
+                if item != 'yum' and item != 'ostree'
+            ]
+        else:
+            non_yum_repo_types = [
+                 item for item in REPO_TYPE.keys() if item != 'yum'
+            ]
         for content_type in non_yum_repo_types:
             with self.subTest(content_type):
                 with self.assertRaises(CLIFactoryError):

--- a/tests/foreman/ui/test_repository.py
+++ b/tests/foreman/ui/test_repository.py
@@ -59,6 +59,7 @@ from robottelo.decorators import (
 )
 from robottelo.decorators.host import skip_if_os
 from robottelo.helpers import get_data_file, read_data_file
+from robottelo.host_info import get_host_os_version
 from robottelo.test import UITestCase
 from robottelo.ui.factory import make_contentview, make_repository, set_context
 from robottelo.ui.locators import common_locators, locators, tab_locators
@@ -922,10 +923,18 @@ class RepositoryTestCase(UITestCase):
 
         @Assert: Dropdown for download policy is not displayed for non-yum repo
         """
-        non_yum_repo_types = [
-            repo_type for repo_type in REPO_TYPE.values()
-            if repo_type != 'yum'
-        ]
+        os_version = get_host_os_version()
+        # ostree is not supported for rhel6 so the following check
+        if os_version.startswith('RHEL6'):
+            non_yum_repo_types = [
+                repo_type for repo_type in REPO_TYPE.values()
+                if repo_type != 'yum' and repo_type != 'ostree'
+            ]
+        else:
+            non_yum_repo_types = [
+                repo_type for repo_type in REPO_TYPE.values()
+                if repo_type != 'yum'
+            ]
         with Session(self.browser):
             for content_type in non_yum_repo_types:
                 self.products.search_and_click(self.session_prod.name)

